go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/containerd/v2
 
-go 1.21
+go 1.22.0
```

Comment:
( This is because of https://github.com/kubernetes/cri-api/blob/v0.30.0-rc.0/go.mod#L5 )